subiquity/server/controllers/filesystem.py

```diff
@@ -731,6 +731,11 @@ async def v2_reformat_disk_POST(self, data: ReformatDisk) \
         self.reformat(self.model._one(id=data.disk_id), data.ptable)
         return await self.v2_GET()
 
+    async def v2_potential_boot_disks_GET(self) -> List[str]:
+        disks = self.get_guided_disks(check_boot=True,
+                                      with_reformatting=False)
+        return [disk.id for disk in disks]
```

Comment:
I wonder if get_guided_disks is the right name for the method called here, I thought it might do a size check. Maybe it should be potential_boot_disks!

Reply:
Right, in practice the size checks are not done by get_guided_disks, but other guided infrastructure.  I like the suggestion to rename it.